### ISSUE TITLE
Something's fishy with Africa/Freetown and timezonedb 2022.2

### ISSUE THREAD

**Issue by goozak** [php/pecl-datetime-timezonedb#2]
[It may already be fixed with 2022.4 which was committed yesterday with some changes in the Africa/Freetown 'section'...]

This code: 
```php
$timeZone = new DateTimeZone('Africa/Freetown');
$trans = $timeZone->getTransitions(time(), strtotime('+1 years', time()));
var_export($trans);
```
 in PHP 8.1.9 (which uses **2022.1**) returns:
```php
array (
  0 => 
  array (
    'ts' => 1664564826,
    'time' => '2022-09-30T19:07:06+0000',
    'offset' => 0,
    'isdst' => false,
    'abbr' => 'GMT',
  ),
)
```

But in 8.1.10 and 8.1.11 (with **2022.2**), it returns:

```php
array (
  0 => 
  array (
    'ts' => 1664565443,
    'time' => '2022-09-30T19:17:23+0000',
    'offset' => -3180,
    'isdst' => false,
    'abbr' => 'LMT',
  ),
  1 => 
  array (
    'ts' => 1672528800,
    'time' => '2022-12-31T23:20:00+0000',
    'offset' => 2400,
    'isdst' => false,
    'abbr' => 'XXX',
  ),
  2 => 
  array (
    'ts' => 1672528800,
    'time' => '2022-12-31T23:20:00+0000',
    'offset' => -3180,
    'isdst' => false,
    'abbr' => 'LMT',
  ),
)
```
Note that each `isdst` values are `false` and the last two `time` are the same...

**Comment by derickr** [comment]
I think this is already fixed, at least in `timezonedb-2022.5`, which you can install through PECL:

```
$ php -dextension=timezonedb -r '$timeZone = new DateTimeZone("Africa/Freetown"); var_dump( $timeZone->getTransitions(time(), strtotime("+1 years", time())));'
Command line code:1:
array(1) {
  [0] =>
  array(5) {
    'ts' =>
    int(1666028856)
    'time' =>
    string(24) "2022-10-17T17:47:36+0000"
    'offset' =>
    int(0)
    'isdst' =>
    bool(false)
    'abbr' =>
    string(3) "GMT"
  }
}
```

```
derick@gargleblaster:~/dev/php/build-php$ php -dextension=timezonedb --ri date

date

date/time support => enabled
timelib version => 2021.16
"Olson" Timezone Database Version => 2022.5
Timezone Database => external
Default timezone => UTC
```

**Comment by goozak** [comment]
Turns out it was indeed already fixed in **2022.4**, which is now included with **PHP 8.1.12**.
